render-test/runner.cpp

```diff
@@ -628,7 +630,23 @@ void resetContext(const TestMetadata& metadata, TestContext& ctx) {
                               .withYSkew(metadata.ySkew));
     map.setDebug(metadata.debug);
     map.getStyle().loadJSON(serializeJsonValue(metadata.document));
-    map.jumpTo(map.getStyle().getDefaultCamera());
+}
+
+LatLng getTileCenterCoordinates(const UnwrappedTileID& tileId) {
+    double scale = (1 << tileId.canonical.z) / util::tileSize;
+    Point<double> tileCenter{tileId.canonical.x + 0.5, tileId.canonical.y + 0.5};
+    return Projection::unproject(tileCenter, scale);
```

Comment:
`Projection::unproject` expects position values to be in world coordinates (or is it called pixel coordinates?) Would it make more sense to multiply `tileCenter` with `1 << tileId.canonical.z` rather than dividing the scale value with `util::tileSize`?